src/components/plots/helpers/changeEmbeddingAxesIfNecessary.js

```diff
@@ -0,0 +1,15 @@
+const changeEmbeddingAxesIfNecessary = (config, method, onUpdate) => {
+  if (!config || !method) return;
+  const { defaultValues: axesDefaultValues, xAxisText, yAxisText } = config?.axes;
```

Comment:
no need for `config?.axes` -- if config is null, the code would have exited on line 2 and never entered here